shablona/shablona.py

```diff
@@ -3,7 +3,15 @@
 from matplotlib import mlab 
 from scipy.special import erf
 import scipy.optimize as opt
+import os.path as op
 
+def get_data_path(absolute = True):
```

Comment:
PEP8: "Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value"